File: VBProg-Youtube/Kifejezések/Kifejezések/Form1.vb
Public Class Form1
    Dim a, c, m, t As Single

    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click
        a = 3
        c = 1
        m = 2
        t = (a + c) / 2 * m
        Label1.Text = t

        a = 6 * (16 - 4 * 3) / (2 + 24 / (5 - 2))
        Label1.Text &= vbNewLine & a
    End Sub
End Class

File: Demeter/NulladikCs/NulladikCs/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace NulladikCs
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            if (button1.Text=="Elrejt")
            {
                label1.Hide();
                button1.Text = "Megmutat";
            }
            else
            {
                label1.Show();
                button1.Text = "Elrejt";
            }
        }

        private void Form1_Resize(object sender, EventArgs e)
        {
            Int32 HolX;
            Int32 HolY;

            HolX = (this.Width - label1.Width) / 2;
            HolY = (this.ClientSize.Height - label1.Height) / 2;

            label1.SetBounds(HolX, HolY, label1.Width, label1.Height);
        }
    }
}
